File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/c/valid.c
/* Fixture — café 日本語 before declarations */
#include <stdio.h>

/* A point. */
struct point {
    int x;
    int y;
};

typedef struct point point_t;

enum color { RED, BLUE };

union value { int i; float f; };

/* Add two ints. */
int add(int a, int b) {
    return a + b;
}

static int *make_ptr(void) { return NULL; }

int prototype_only(int a);

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/javascript/valid.js
// Fixture: café — naïve 日本語 before any declaration
import { helper } from "./helper.js";

/** Adds two numbers. */
export function add(a, b) {
  return helper(a) + b;
}

class Counter {
  constructor() { this.n = 0; }
  /** Increment. */
  increment() { this.n += 1; }
}

export const answer = 42;
